src/colvarscript.cpp

```diff
@@ -480,13 +481,36 @@ int colvarscript::proc_bias(colvarbias *b, int objc, unsigned char *const objv[]
     if (subcmd == "count") {
       int index;
       if (!(std::istringstream(param) >> index)) {
-        result = "bin_count: error parsing bin index";
+        result = "bias count: error parsing bin index";
         return COLVARSCRIPT_ERROR;
       }
       result = cvm::to_str(b->bin_count(index));
       return COLVARS_OK;
     }
 
+    // Get ABF gradient
+    if (subcmd == "gradient") {
+      int index;
+      if (!(std::istringstream(param) >> index)) {
+	result = "bias gradient: error parsing bin index";
+	return COLVARSCRIPT_ERROR;
+      }
+      result = cvm::to_str(-b->bin_gradient(index));
```

Comment:
Legal syntax, but confusing to read.

Reply:
Is it the minus sign that you find confusing?

Reply:
Yes, what about some parenthesis?  Unlike in Python, unnecessary parentheses are not considered bad style :-)